Sources/MongoSwift/MongoClient.swift

```diff
@@ -260,6 +307,9 @@ public class MongoClient {
             mongoc_database_set_write_concern(db, wc._writeConcern)
         }
 
-        return MongoDatabase(fromDatabase: db, withClient: self)
+        let encoder = BSONEncoder(copies: self.encoder, options: options)
```

Comment:
follow-on work: `MongoDatabase` should be responsible for setting all these values up, move concerns to the `init` there. It should just be `return MongoDatabase(withClient: self, withOptions: options)`

Reply:
I actually think this is worth updating now. There's no reason to pass it this way if the database can just get them through the client.

Reply:
I now see why this is future work. Filed [SWIFT-377](https://jira.mongodb.org/browse/SWIFT-377) for it.